resources/shaders/lighting.hlsl

```diff
@@ -121,7 +121,13 @@ float3 shade_pixel(float3 pos, float3 V, float3 albedo, float metallic, float ro
 	[unroll]
 	for (uint i = 0; i < light_count; i++)
 	{
-		res += shade_light(pos, V, albedo, normal, metallic, roughness, lights[i], rand_seed, depth);
+		if(shadows == 1)
```

Comment:
Why this change seems to be irrelevant to ao.

Reply:
During development I was asked to add a toggle for AO, while implementing that I also implemented the same thing for shadows and reflections as it would save work in the end.